components/email/src/Email.svelte

```diff
@@ -1432,11 +1490,33 @@
                     <div class="message-body">
                       {#if _this.clean_conversation && message.conversation}
                         {@html DOMPurify.sanitize(message.conversation)}
-                      {:else if message.body}
+                      {:else if message && typeof message.body === "string"}
                         <nylas-message-body
                           {message}
+                          body={message.body}
                           on:downloadClicked={handleDownloadFromMessage}
                         />
+                        <!-- If a thread is being passed manually and there is no body, 
+                          we will keep loading, so the below is our fallback -->
+                      {:else if !!_this.thread && !_this.thread_id && click_action != "mailbox"}
```

Comment:
thank you for the comment! Makes a lot more sense now that I realize thread = local passed thread